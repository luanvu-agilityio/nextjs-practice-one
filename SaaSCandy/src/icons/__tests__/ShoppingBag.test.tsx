import { render } from '@testing-library/react';
import { ShoppingBagIcon } from '../ShoppingBag';

describe('AddressIcon', () => {
  it('matches snapshot', () => {
    const { container } = render(<ShoppingBagIcon />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
