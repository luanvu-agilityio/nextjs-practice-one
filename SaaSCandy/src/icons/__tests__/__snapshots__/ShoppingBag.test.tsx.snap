// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`AddressIcon matches snapshot 1`] = `
<svg
  fill="none"
  height="60"
  viewBox="0 0 60 60"
  width="60"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="30"
    cy="30"
    fill="#FFF4CE"
    r="30"
  />
  <mask
    height="60"
    id="mask0_47_5413"
    maskUnits="userSpaceOnUse"
    width="60"
    x="0"
    y="0"
  >
    <circle
      cx="30"
      cy="30"
      fill="#FFF4CE"
      r="30"
    />
  </mask>
  <g
    mask="url(#mask0_47_5413)"
  >
    <circle
      cx="14.7764"
      cy="52.7317"
      fill="#FFDFAF"
      r="30"
    />
  </g>
  <path
    d="M24.1665 30.8643C24.324 30.8643 24.4751 30.9268 24.5864 31.0381C24.6978 31.1495 24.7603 31.3006 24.7603 31.4581V37.2911C24.7603 37.4485 24.6977 37.5997 24.5864 37.711C24.4751 37.8223 24.324 37.8848 24.1665 37.8848C24.009 37.8848 23.8579 37.8223 23.7466 37.711C23.6353 37.5997 23.5728 37.4485 23.5728 37.2911V31.4581C23.5728 31.3006 23.6352 31.1495 23.7466 31.0381C23.8579 30.9268 24.009 30.8643 24.1665 30.8643ZM29.9995 30.8643C30.1569 30.8643 30.3081 30.9269 30.4194 31.0381C30.5308 31.1495 30.5933 31.3006 30.5933 31.4581V37.2911C30.5933 37.4485 30.5308 37.5996 30.4194 37.711C30.3081 37.8223 30.157 37.8848 29.9995 37.8848C29.8422 37.8847 29.6909 37.8223 29.5796 37.711C29.4684 37.5997 29.4058 37.4484 29.4058 37.2911V31.4581C29.4058 31.3007 29.4683 31.1495 29.5796 31.0381C29.6909 30.9269 29.8422 30.8644 29.9995 30.8643ZM35.8335 30.8643C35.9909 30.8644 36.1421 30.9269 36.2534 31.0381C36.3647 31.1495 36.4272 31.3007 36.4272 31.4581V37.2911C36.4272 37.4484 36.3646 37.5997 36.2534 37.711C36.1421 37.8223 35.9909 37.8847 35.8335 37.8848C35.676 37.8848 35.5249 37.8223 35.4136 37.711C35.3022 37.5996 35.2397 37.4485 35.2397 37.2911V31.4581C35.2397 31.3006 35.3022 31.1495 35.4136 31.0381C35.5249 30.9269 35.6761 30.8643 35.8335 30.8643Z"
    fill="#EE7B11"
    stroke="#EE7B11"
  />
  <path
    d="M27.0835 16.281H32.9165C33.3051 16.2809 33.6858 16.3916 34.0142 16.5994C34.3426 16.8072 34.6056 17.1036 34.772 17.4548L34.9058 17.738L35.2202 17.741C36.2178 17.7483 36.9103 17.796 37.5093 18.03C38.25 18.3195 38.8942 18.8117 39.3687 19.45C39.8347 20.0791 40.067 20.9003 40.4204 22.199L41.3354 25.5583L41.394 25.7722L41.5933 25.8728C42.1007 26.1293 42.5454 26.4945 42.896 26.9421V26.9412C43.6885 27.9566 43.8543 29.185 43.6997 30.7175C43.5468 32.2114 43.0741 34.1092 42.4644 36.5496L42.4253 36.7029L42.4243 36.7068C42.038 38.2533 41.735 39.466 41.3765 40.4041L41.3755 40.405C41.0068 41.3733 40.5603 42.112 39.8501 42.6648L39.8491 42.6658C39.14 43.2213 38.3158 43.476 37.2866 43.5984H37.2856C36.2882 43.7179 35.0379 43.7185 33.4448 43.7185H26.5552C24.962 43.7185 23.7098 43.7176 22.7134 43.5994H22.7144C21.8136 43.4896 21.0687 43.2819 20.4204 42.8601L20.1489 42.6667C19.4393 42.1121 18.991 41.3728 18.6226 40.405C18.2651 39.4665 17.9626 38.2533 17.5747 36.7078H17.5737L17.5376 36.5554L17.5366 36.5515L17.1069 34.8201C16.7091 33.1864 16.4139 31.8382 16.3003 30.7166C16.1443 29.1871 16.3104 27.9567 17.103 26.9412L17.1021 26.9402C17.4529 26.4928 17.8991 26.1289 18.4067 25.8728L18.606 25.7722L18.6636 25.5583L19.5796 22.199C19.9314 20.9005 20.1636 20.08 20.6304 19.4529V19.4519C21.0453 18.8927 21.5909 18.4455 22.2173 18.1482L22.4907 18.03C23.0899 17.7959 23.7811 17.7478 24.7808 17.739L25.0933 17.7371L25.228 17.4539C25.3945 17.1028 25.6574 16.806 25.9858 16.5984C26.3142 16.3909 26.6951 16.2808 27.0835 16.281ZM26.6235 26.2185C24.0248 26.2185 22.1414 26.2199 20.7397 26.4216H20.7388C19.3539 26.6226 18.5603 27.0053 18.0396 27.6716C17.5185 28.3384 17.3404 29.2015 17.4819 30.5955C17.625 32.0049 18.0816 33.8321 18.7104 36.3533V36.3542C19.0625 37.7574 19.3273 38.8168 19.6099 39.6433L19.7329 39.9832C20.0587 40.8386 20.404 41.3585 20.8813 41.7312C21.357 42.1021 21.9435 42.3106 22.854 42.4187V42.4197C23.7788 42.5305 24.968 42.531 26.6235 42.531H33.3765C35.0317 42.531 36.2189 42.5305 37.145 42.4197H37.146C38.0539 42.3102 38.6424 42.1026 39.1187 41.7312L39.1196 41.7302C39.5948 41.3578 39.9399 40.8391 40.2671 39.9841V39.9832C40.598 39.1119 40.8877 37.9578 41.2886 36.3542C41.919 33.8327 42.3739 32.0059 42.5171 30.5964C42.6513 29.2888 42.502 28.4498 42.0532 27.8005L41.9595 27.6736C41.4385 27.0056 40.6452 26.6225 39.2593 26.4216H39.2583C37.8567 26.2199 35.9749 26.2185 33.3765 26.2185H26.6235ZM34.771 19.2146C34.4395 19.9077 33.7331 20.3854 32.9155 20.3855H27.0815C26.2672 20.3854 25.5607 19.908 25.229 19.2146L25.0913 18.9265L24.772 18.9294C23.8246 18.9395 23.3434 18.9723 22.9233 19.1365H22.9224C22.4557 19.3191 22.042 19.6132 21.7173 19.9919L21.5835 20.1599C21.4118 20.3911 21.2895 20.6523 21.1587 21.0339C21.0941 21.2222 21.0252 21.448 20.9458 21.7234L20.6714 22.7107L20.1548 24.6023L19.9565 25.3298L20.7036 25.2292C22.1766 25.032 24.0919 25.031 26.5317 25.031H33.4663C35.9078 25.031 37.8234 25.032 39.2964 25.2292L40.0435 25.3298L39.8452 24.6023L39.3286 22.7107C39.1174 21.9364 38.9691 21.4102 38.8403 21.0339C38.7098 20.6528 38.5883 20.3913 38.4165 20.1599C38.0746 19.6994 37.6107 19.3456 37.0776 19.1365H37.0767C36.6566 18.9723 36.1754 18.9395 35.228 18.9294L34.9087 18.9265L34.771 19.2146ZM27.0835 17.4685C26.883 17.4685 26.6894 17.538 26.5356 17.6638L26.4722 17.7214C26.3101 17.8835 26.2183 18.1036 26.2183 18.3328C26.2183 18.562 26.3101 18.782 26.4722 18.9441C26.6343 19.1062 26.8542 19.198 27.0835 19.198H32.9165C33.1458 19.198 33.3657 19.1062 33.5278 18.9441C33.6899 18.782 33.7808 18.562 33.7808 18.3328C33.7807 18.1036 33.6899 17.8835 33.5278 17.7214C33.3657 17.5594 33.1457 17.4685 32.9165 17.4685H27.0835Z"
    fill="#EE7B11"
    stroke="#EE7B11"
  />
</svg>
`;
